webpack.config.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.ts',
+  output: {
+    library: 'LookupService',
```

Comment:
Since this library has two services, the default name of Webpack bundle cannot be guessed correctly in this case. In case if anyone wants to use it in Webpack, `LookupService` is misleading. You can add `"main-service": "ServiceDirectory"` to the generator arguments in `synth.py` to change this to `ServiceDirectory`.